Makefile

```diff
@@ -3,13 +3,13 @@ dbt-trino-tests:
 	./docker/dbt/build.sh
 	./docker/init_trino.bash
 	pip install -r dev_requirements.txt
-	tox
+	tox || ./docker/remove_trino.bash
```

Comment:
what was happening in case of a failure before?

Reply:
Docker container was up and running